docbook/xslt20-stylesheets
# Replace jython.jar with one including jython python lib modules

The packaged jython.jar file does not include the python library modules, such as `re` and `datetime`. When using the jython.jar to load jython and pygments to highlight <programlisting>s, `import pygments` fails due jython not being able to find a missing dependency on the `re` module. Pointing jython at a local Lib repository is error prone as the installed jython libs must match the jython.jar version used to build the docbook-xsl2-saxon.jar file.

To test:

```
java -jar tools/lib/jython.jar
>>> import re
```

Result:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named re
```

Expected:
`re` module loads successfully

This is pull request is just a repackaging of the previous jython.jar version to include the python libraries in addition to the core jython files.
- The existing jython.jar package does not include python library
  modules. This causes source code highlighting with pygments to fail
  due to missing dependencies on the python re module.
